h2o-core/src/main/java/water/util/ArrayUtils.java

```diff
@@ -2219,4 +2219,56 @@ private static boolean isPresentInAllWords(String prefix, String[] words) {
 
     return p;
   }
+
+  /*
+   * Linear interpolation values in the array with Double.NaN values.
+   * The interpolation always starts from 0. 
+   * The last element of array cannot be Double.NaN.
+   *
+   * @param array input array with Double.NaN values
+   */
+  public static void interpolateLinear(double[] array){
+    assert array.length > 0 && !Double.isNaN(array[array.length-1]);
```

Comment:
Can we have some message here? I faced this assert before when I had some input parameter not correct, but I cannot reproduce it in the new version.

Reply:
There was a bug, so I change the code. That maybe why you are not able to reproduce it. What do you set into input parameters? 

I add a message there, no problem.